partcad-cli/src/partcad_cli/click/commands/convert.py

```
@@ -0,0 +1,37 @@
+    required=False,
+)
+@click.pass_obj
+def cli(ctx: Context, part_name: str, target_format: str):
+    """
+    CLI command to convert a part to a new format or resolve an enrich part.
+    """
+    project_name, object_name = pc_utils.resolve_resource_path(ctx.get_current_project_path(), part_name)
+    project = ctx.get_project(project_name)
+
+    if not project:
+        raise click.UsageError(f"Failed to find project '{project_name}'.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)